rest_sensors.cpp

```diff
@@ -2510,13 +2510,23 @@ void DeRestPluginPrivate::checkSensorStateTimerFired()
                 }
                 else if (!item && sensor->modelId() == QLatin1String("lumi.vibration.aq1") && sensor->type() == QLatin1String("ZHAVibration"))
                 {
+                    //Set tiltangle to 0 by defaut but without QueueEvent
+                    item = sensor->item(RStateTiltAngle);
```

Comment:
Best move this block into the `if (item && item->toBool())` so that it gets executed only when `state.vibration` is reset to `false`.  Then you can issue the event for `state.tiltangle` without getting false positives.